keyboards/idobo/keymaps/Revok75/keymap.c

```diff
@@ -0,0 +1,90 @@
+		KC_GESC,  KC_1,     KC_2,     KC_3,     	KC_4,     	KC_5, 	KC_6,   KC_7,   				KC_8,     KC_9,     KC_0,     KC_MINS,    KC_EQL,     LGUI(KC_X),  KC_BSPC, \
+		KC_TAB,   KC_Q,     KC_W,     KC_E,     	KC_R,     	KC_T, 	KC_Y,   KC_U,   				KC_I,     KC_O,     KC_P,     KC_LCBR,    KC_RCBR,    LGUI(KC_C),  LGUI(KC_V), \
+		KC_CAPS,  KC_A,     KC_S,     KC_D,     	KC_F,     	KC_G, 	KC_H,   KC_J,   				KC_K,     KC_L,     KC_SCLN,  KC_QUOT,    LGUI(KC_R), KC_NO,       KC_ENT, \
+		KC_LSFT,  KC_Z,     KC_X,     KC_C,     	KC_V,     	KC_B, 	KC_N,   KC_M,   				KC_COMM,  KC_DOT,   KC_SLSH,  KC_BSLS,    KC_RSFT,    KC_UP,       LGUI(KC_S), \
+		KC_LCTL,  KC_LALT,  KC_LGUI,	LGUI(KC_F), LGUI(KC_T), KC_SPC, KC_NO, 	LALT(KC_SPACE),	MO(_FN), 	KC_RGUI,  KC_RALT,  KC_RCTL,    KC_LEFT,    KC_DOWN,     KC_RGHT \
+	),
+
+	 /* QWERTY-Win
+	  * .--------------------------------------------------------------------------------------------------------------------------------------.
+	  * | ESC    | 1      | 2      | 3      | 4      | %      | 6     | 7      | 8      | 9      | 0      | -       | =    | Cut    | BACKSP   |
+	  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+	  * | TAB    | Q      | W      | E      | R      | T      | Y     | U      | I      | O      | P      | {      | }     | Copy   | Paste    |
+	  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+	  * | CAP LK | A      | S      | D      | F      | G      | H     | J      | K      | L      | ;      | '      | ENTER  | ENTER | REF      |
+	  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+	  * | LSHIFT | Z      | X      | C      | V      | B      | N     | M      | ,      | .      | /      | \      | RSHIFT | UP    | SAVE     |
+	  * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+	  * | LALT   | LGUI   | LALT   |  FIND  | CMD T	 | SPACE | SPACE  | START  | FN-MO  | RALT   | RGUI   | RCTRL  | LEFT  | DOWN   | RIGHT    |
+	  * '--------------------------------------------------------------------------------------------------------------------------------------'
+	  */
+
+  [_QW_W] = LAYOUT_ortho_5x15( /* QWERTY WIN*/
+  	KC_GESC,  KC_1,     KC_2,     KC_3,     		KC_4,     	KC_5, 	KC_6,   KC_7,   				KC_8,     KC_9,     KC_0,     KC_MINS,     KC_EQL,     LCTL(KC_X),  KC_BSPC, \
+  	KC_TAB,   KC_Q,     KC_W,     KC_E,     		KC_R,     	KC_T, 	KC_Y,   KC_U,   				KC_I,     KC_O,     KC_P,     KC_LCBR,     KC_RCBR,    LCTL(KC_C),  LCTL(KC_V), \
+  	KC_CAPS,  KC_A,     KC_S,     KC_D,     		KC_F,     	KC_G, 	KC_H,   KC_J,   				KC_K,     KC_L,     KC_SCLN,  KC_QUOT,     LCTL(KC_R), KC_NO,       KC_ENT}, \
```

Comment:
Here's the error that's causing your Idobo keymap to not compile. The closing curly brace on this line didn't get removed.

```suggestion
  	KC_CAPS,  KC_A,     KC_S,     KC_D,     		KC_F,     	KC_G, 	KC_H,   KC_J,   				KC_K,     KC_L,     KC_SCLN,  KC_QUOT,     LCTL(KC_R), KC_NO,       KC_ENT, \
```